src/shipit_signoff/settings.py

```diff
@@ -11,6 +11,7 @@
 import shipit_signoff.util
 
 
+LOCAL_AUTH = bool(os.environ.get('LOCAL_AUTH', False))
```

Comment:
Should we make LOCAL_AUTH be true by when in DEBUG mode. DEBUG variable is set to true (aka DEBUG mode) when you run ``./please run <project>``.